### PR TITLE
Updated recipe

### PR DESCRIPTION
**Pull request by melund**
Updated recipe  with more info (inspired by the spyder feedstock recipe https://github.com/conda-forge/staged-recipes/pull/477 )

Added …
- Start menu entries on Win ( Seems to work nicely with the newest version of conda)
- Integration with Anaconda Navigator (the replacement for Anaconda launcher) 
- More detailed description in the recipe. 
- updates from `conda smithy rerender`

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -35,17 +35,28 @@ test:
...
 
 app:
   entry: xonsh
   icon: xonsh.png
+  summary: xonsh is a Python-ish, BASHwards-facing shell.
+  type: desk
 
 about:
   home: http://xon.sh
   license: BSD
```
Comment (jakirkham) [inline recipe/meta.yaml]: Could you please specify the version of BSD?

Comment (jakirkham) [inline recipe/meta.yaml]: Looks like this is BSD 2-Clause.

Comment (melund) [inline recipe/meta.yaml]: Hmm. I have no clue... @scopatz ?

Comment (jakirkham) [inline recipe/meta.yaml]: That's what I found [here]( https://github.com/scopatz/xonsh/blob/master/license ).

Comment (scopatz) [inline recipe/meta.yaml]: Yeah, it is 2-clause.

**Comment by scopatz** [comment]
Looks great to me! Thanks!

**Pull request merged**